android_mobile/src/main/java/com/alexstyl/specialdates/donate/DonateActivity.java

```diff
@@ -81,13 +83,13 @@ public void onClick(View v) {
             @Override
             public void run() {
                 scrollToDonate();
-
             }
 
             private void scrollToDonate() {
                 CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) appBarLayout.getLayoutParams();
                 AppBarLayout.Behavior behavior = (AppBarLayout.Behavior) params.getBehavior();
-                behavior.onNestedFling(coordinator, appBarLayout, null, 0, 50, true);
+                if (behavior != null)
```

Comment:
I see. if that is the case, then the null check is not really needed. The user is not going to be navigated to this Activity unless they have Play Services any way (see the the [MainNavigator](https://github.com/alexstyl/Memento-Calendar/blob/develop/android_mobile/src/main/java/com/alexstyl/specialdates/ui/activity/MainNavigator.java#L45) class)